benchmarking/runs/social/varied_num_enemies.py

```diff
@@ -0,0 +1,187 @@
+from typing import Dict, List
+
+import typer
+
+from api.ai.interfaces.algorithm_config import (
+    WeightAlgorithmConfig,
+    SocialAlgorithmConfig,
+    RandomAlgorithmConfig,
+    AlgorithmConfig,
+)
+from api.models.enums import AlgorithmType
+from benchmarking.data.simulated_data.mock_student_provider import (
+    MockStudentProviderSettings,
+    MockStudentProvider,
+)
+from benchmarking.evaluations.graphing.graph_metadata import GraphData, GraphAxisRange
+from benchmarking.evaluations.graphing.line_graph import line_graph
+from benchmarking.evaluations.graphing.line_graph_metadata import LineGraphMetadata
+from benchmarking.evaluations.interfaces import TeamSetMetric
+from benchmarking.evaluations.metrics.average_social_satisfied import (
+    AverageSocialSatisfaction,
+)
+from benchmarking.evaluations.metrics.utils.team_calculations import *
+from benchmarking.evaluations.scenarios.give_the_people_what_they_want import (
+    GiveThePeopleWhatTheyWant,
+)
+from benchmarking.runs.interfaces import Run
+from benchmarking.simulation.insight import Insight, InsightOutput
+from benchmarking.simulation.simulation_set import SimulationSet, SimulationSetArtifact
+from benchmarking.simulation.simulation_settings import SimulationSettings
+
+
+class VariedNumEnemiesSocialRun(Run):
+    @staticmethod
+    def start(num_trials: int = 10, generate_graphs: bool = True):
```

Comment:
we said it should default to false but we will fix these all at once when we decide on the default num trials, so this is a nit;